ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -308,7 +229,8 @@ def create_instance_in_clusterlogging(sc_name=None):
     inst_data = templating.load_yaml_to_dict(constants.CL_INSTANCE_YAML)
     inst_data['spec']['logStore']['elasticsearch']['storage']['storageClassName'] = sc_name
     inst_data['spec']['logStore']['elasticsearch']['storage']['size'] = "200Gi"
```

Comment:
can this be made configurable parameter, probably in a later patch?

Reply:
@keesturam  I will take it up in the later patch